src/diffusers/loaders/ip_adapter.py

```diff
@@ -46,7 +48,6 @@ class IPAdapterMixin:
     def load_ip_adapter(
         self,
         pretrained_model_name_or_path_or_dict: Union[str, Dict[str, torch.Tensor]],
-        subfolder: str,
```

Comment:
This is a breaking change, can we leave as it was here?

Reply:
done as you suggested!